### ISSUE TITLE
Port to Ubuntu Touch

### ISSUE THREAD

**Issue by ktatar156** [mapsme/omim#2358]
Hello,

Is there any possibility that anyone could port that great app to Ubuntu Touch? I think we can create crowd funding campaign that could successful get some necessary resources.

**Comment by deathbaba** [comment]
Have you tried to compile and run it? If QT works on Ubuntu Touch, the basic version should fly.

**Comment by ktatar156** [comment]
No; I'm not developer, so my knowledge is little. But I think that there are some parts that need to be adjusted to hit Ubuntu Touch (like GUI or usage of location (because of some restriction while app is unfocused)).

**Comment by mitchellreese** [comment]
Yep, would love to see this also!

**Comment by utack** [comment]
Kind of OT, but maybe Android Auto should have higher priority?

**Comment by keunes** [comment]
Given that Ubuntu Touch/Phone has been ditched by Canonical, I guess this can be closed - unless people still want to port it themselves?

**Comment by jonnius** [comment]
Ubuntu Touch is pretty much alive and driven bei UBports nowadays.
